src/NotifyIconWpf/Interop/WinApi.cs

```diff
@@ -87,5 +87,15 @@ internal static class WinApi
 
         [DllImport(User32, SetLastError = true)]
         public static extern bool GetCursorPos(ref Point lpPoint);
+
```

Comment:
I'm all for clean code and reuse, but somethings we should not over engineer code.

I actually don't see any re-use for this, unless there is no need for this elsewhere I would suggest to just have this inside the switch statement with the WindowsMessages.WM_CONTEXTMENU